### ISSUE TITLE
Dependency with graphviz library

### ISSUE THREAD

**Issue by jmhorcas** [diverso-lab/bdd_metamodel#23]
The BDD plugin has a dependency not specified in the requirements.
The library graphviz is required in order to write the BDD to an image (png, svg).

This dependency needs to be installed as follows:
- python package: `pip install graphviz`
- Graphviz software: `sudo apt install graphviz`

Some tips?

**Comment by Virako** [comment]
this dependency should add in setup.py, inside install_requires array how 'graphviz~=0.20'

**Comment by jmhorcas** [comment]
OK. I've added such dependency. However, without the software Graphviz installed it would still fail.

**Comment by Virako** [comment]
That's true, I believe that adding it in the installation guide is the only thing we can do.
